exercises/matrix/example.exs

```diff
@@ -0,0 +1,63 @@
+defmodule MatrixStruct do
```

Comment:
It initially was called Matrix, but that failed CI because the saddle-points exercise has its module named Matrix for some reason. That should probably be changed, so this can be just Matrix, but if you have a better name in the meantime, I'm happy to change it.